.github/workflows/main.yml

```diff
@@ -167,6 +167,15 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v3
 
+      - name: git checkout fuzz seed corpus
+        uses: actions/checkout@v3
+        with:
+          repository: morehouse/lnd-fuzz-seed-corpus
```

Comment:
The repo has been created: https://github.com/lightninglabs/lnd-fuzz

I think we can update this now to point to it instead.

Reply:
Done.